Gemfile

```diff
@@ -6,6 +6,7 @@ gemspec
 
 group :development, :test do
   gem 'debug'
+  gem 'rack-test'
   gem 'foreman'
   gem 'rerun'
```

Comment:
I guess the rules changed since whoever originally wrote this test kit, but we no longer require `foreman` and `rerun` in development dependencies, and ask users to install it globally via `gem install foreman rerun`. Can you remove these two lines please